app/code/Stanislavz/PhpOop/ViewModel/DependencyInjection.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Stanislavz\PhpOop\ViewModel;
+
+class DependencyInjection implements \Magento\Framework\View\Element\Block\ArgumentInterface
+{
+    public $stringParam;
```

Comment:
What the reason of having these values public? If they made public for reflection example, please leave the comment about it. Also no usage of these properties has found.